### PR TITLE
Remove tests/manage/pv_services/test_reclaim_policy.py

### PR DESCRIPTION
**Pull request by sidhant-agrawal**
Fixes: #2450 
~~Test now uses new factory fixture and default cephblockpool~~
Disabled tests `OCS-383` and `OCS-384`  as they are marked Inactive in polarion

Signed-off-by: Sidhant Agrawal <sagrawal@redhat.com>

### CODE REVIEW

**Comment by RazTamir** [comment]
Hi @Sidhant,

Any chance the tests marked as inactive by mistake? 
I wonder why we are removing coverage? 

Thanks

**Comment by sidhant** [comment]
> Hi @sidhant,
> 
> Any chance the tests marked as inactive by mistake?
> I wonder why we are removing coverage?
> 
> Thanks

Sorry, wrong sidhant :-)

**Comment by sidhant-agrawal** [comment]
@RazTamir 
The tests OCS-383 and OCS-384 have been marked duplicates of tests like OCS-525, OCS-533, OCS-538

OCS-525 & 533 are automated in `tests/manage/pv_services/test_dynamic_pvc_accessmodes_with_reclaim_policies.py`
OCS 538 - This test is marked inactive in polarion, and covered by OCS-751 in `tests/manage/pv_services/test_raw_block_pv.py`

File: tests/manage/pv_services/test_reclaim_policy.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ class TestReclaimPolicy(ManageTest):
     """
 
     @pytest.mark.polarion_id("OCS-383")
-    def test_reclaim_policy_retain(self):
+    def reclaim_policy_retain(self):
```
Comment (vasukulkarni) [inline tests/manage/pv_services/test_reclaim_policy.py:70]: might be worth deleting than if its already duplicated ?

Comment (sidhant-agrawal) [reply tests/manage/pv_services/test_reclaim_policy.py:70]: I wasn't sure if deletion is preferred so just disabled them.
We can delete if that is an agreement for that.
@ebenahar @keesturam

Comment (vasukulkarni) [reply tests/manage/pv_services/test_reclaim_policy.py:70]: @sidhant-agrawal @ebenahar any update here? Deletion is preferred for *dead* code, git still shows the history if *at all* needed, lets keep the code that is in use.

Comment (sidhant-agrawal) [reply tests/manage/pv_services/test_reclaim_policy.py]: Now removing the `test_reclaim_policy.py`

**Pull request merged**